settings.coffee

```diff
@@ -169,7 +171,7 @@ settings =
 	# Should javascript assets be served minified or not. Note that you will
 	# need to run `grunt compile:minify` within the web-sharelatex directory
 	# to generate these.
-	useMinifiedJs: true
+	useMinifiedJs: false
```

Comment:
Is there a particular problem with the minification step? Most CE users will be on LANs, so it's not too bad, but if we can keep it, it would save people some bandwidth.

Reply:
I'll check this again. I think I disabled it before updating how `web` is built in community edition.

Reply:
I spent some time trying to build the minified version of Overleaf, unsuccessfully. 

For Overleaf community we're currently using a slightly different build process in which we're calling straight `make compile_full` instead of `make build` (which doesn't work). Internally `make build` is supposed to run the project's `Dockerfile` which in turn runs `make compile_full`, but seems minification is not produced in the context of Community build.

For the time being I think it make sense to add a separate ticket and address it separately, since it's not a showstopper. 

Any thoughts @jdleesmiller ?

Reply:
It might be worth cc'ing @paulojreis on minification